uploader.js

```diff
@@ -1,17 +1,28 @@
 const {Storage} = require('@google-cloud/storage');
+const path = require('path');
+
 const storage = new Storage();
-const filename = process.argv.slice(2);
 
+if(process.argv.length < 3){
+    console.log("Need file to upload");
+    process.exit(1);
+}
+const fileName = process.argv[2];
 const bucketName = process.env.BUCKET_NAME;
+const uploadDir = process.env.UPLOAD_DIR;
+
+var destination = fileName;
```

Comment:
`var` id deprecated, use `let` instead
```suggestion
let destination = fileName;
```